### ISSUE TITLE
Stopping criteria

### ISSUE THREAD

**Issue by wjb123** [KaiyangZhou/pytorch-vsumm-reinforce#9]
As you said in the paper, there is a description on stopping criteria. "For all our models, we stop training after K consecutive epochs with descending summarization F-score on the validation set. We set K = 5", but I can not see any clue on this strategy from your code. Moreover, I don't  see validation set in your code. Can you make some explanation ?

**Comment by KaiyangZhou** [comment]
the released code is not exactly the same as the one implemented for the paper

**Comment by wjb123** [comment]
So how do you choose the validation set ? For SumMe dataset, there are 20 videos for training and 5 videos for testing.

**Comment by wjb123** [comment]
I find the test F-score (on the test set) is the same when the model is training on different epochs,  Can you tell me the reason ?
